### DISCUSSION TITLE
Exclude data from export (influxdb2)

### DISCUSSION THREAD

**Discussion by BenRoe** [nicolargo/glances#2579] [Q&A] [answered]
Hello,
i have glances and influxdb2 running with docker compose.
I to keep the database small as possible, i wanna export only cpu, docker and cpu temp.
Is there a way to do that?

**Comment by nicolargo** [comment, marked as answer]
Something like that:

    glances --disable-plugin all --enable-plugin cpu,docker,sensors -t 5 --export influxdb

**Reply by BenRoe** [reply]
Thank you. That’s will reduce the data a lot.
I also habe many interfaces.
Is it possible to include only eth0?
![image](https://github.com/nicolargo/glances/assets/643834/17d8027e-6e4a-41c0-bd0f-815cc4a70302)

**Comment by nicolargo** [comment]
Yes it is possible, just have to configure the **hide** option in the **network** section of the Glances configuration file: https://glances.readthedocs.io/en/latest/aoa/network.html

**Comment by BenRoe** [comment]
I was looking for a list with all plugin names, that i can use with the parameter `--enable-plugin`?
